pallets/phala/src/compute/computation.rs

```diff
@@ -123,11 +123,11 @@ pub mod pallet {
 		}
 	}
 
-	/// The state of a miner
+	/// The state of a worker
 	#[derive(Encode, Decode, TypeInfo, Clone, PartialEq, Eq, RuntimeDebug)]
-	pub struct MinerInfo {
-		/// The current state of the miner
-		pub state: MinerState,
+	pub struct WorkerInfo {
```

Comment:
Better "SessionInfo" or "ComputeInfo". In this file, "worker" usually refers to the physical worker, and "miner" refers to the mining session.